.github/dockerfiles/ov_build/ubuntu_22_04_riscv/Dockerfile

```diff
@@ -62,10 +64,14 @@ RUN echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy main restricte
 
 RUN dpkg --add-architecture riscv64 && \
     apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/riscv64-sources.list && \
-    apt-get install -y --no-install-recommends libpython3-dev:riscv64
+    apt-get install -y --no-install-recommends libpython3-dev:riscv64 && \
+    apt-get install -y --no-install-recommends apt-utils && \
+    apt-get -y install  curl && apt-get install libgomp1:riscv64 && \
+    apt-get install libatomic1:riscv64
```

Comment:
```suggestion
       apt-get install \
           libpython3-dev:riscv64 \
           libgomp1:riscv64 \
           libatomic1:riscv64
```

Reply:
apt-utils should be install on the arch independent step - https://github.com/mryzhov/openvino/blob/gha/smart_ci_snippets/.github/dockerfiles/ov_build/ubuntu_22_04_riscv/Dockerfile#L22